### PR TITLE
check the code.

### PR DESCRIPTION
**Pull request by WALWALASAFI**
please review code  changes and approve

### CODE REVIEW

**Comment by WALWALASAFI** [comment]
Hi sir, 
i hope ur fie, kindly check my project
@ahzamir
@ahzia

**Comment by WALWALASAFI** [comment]
the code checked correctly in free code camp and have no linter error.
un7952210_20220919_lf_1683_
is there anything remind for approving the project dear sir,
@ahzamir
@ahzia 

thanks in advance!

**Comment by WALWALASAFI** [comment]
the code have been checked and accepted in free code camp. Also have no linter error.
is there anything remind for approving the project.
dear sir, 
 @ahzamir
@ahzia
thanks in advance!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        She is the daughter of Ziauddin Yousafzai and Toor Pekai Yousafzai.
+        Her family is Sunni Muslim[5] of Pashtun ethnicity, belonging to the Yusufzai tribe.
+        The family did not have enough money for a hospital birth and Yousafzai was born at home with the help of neighbours.
+        She was given her first name Malala (meaning "grief-stricken") after Malalai of Maiwand, 
+        a famous Pashtun poet and warrior woman from southern Afghanistan.
+        At her house in Mingora, she lived with her two younger brothers, Khushal and Atal, her parents, Ziauddin and Tor Pekai, and two chickens.
+       </p>
+       </div> 
+       <a href="https://en.wikipedia.org/wiki/Malala_Yousafzai" id="tribute-link" target="_blank" rel="noopener">Learn more about Malala-Yousafzai</a>
+ </main>
```
Comment (ahzamir) [inline index.html:54]: @WALWALASAFI 

- Although your code passes the feecodecamp and meets all the requirements , butyour code is still not clean:
- And for cleaning your code:
  - At the bottom right side of your vscode click on `Spaces` then click on `Index using spaces` and choose 2 from there.
  - Then according to the lines on the screen all child elements should be 2 spaces or one line ahead of their parent element and the closing tag should be in the same line as the opening tag!
  
  - Your body spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

File: style.css
Comment (ahzamir) [inline style.css:1]: - Add some more design to your page and elements so it looks more amazing to share.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @WALWALASAFI 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although your code passes all the checks on freecodecamp website, but your HTML code is not clean please check the comments below for more details.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Tribute Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by WALWALASAFI** [comment]
hi dear sir,
i have done correct. what u said.
and about style.css i linked it with my html  however it commited and pushed but doesnt apper on page. when .
this is my style.css.
style .css=
@ahzamir

**Review by ahzamir — CHANGES_REQUESTED** [review]
@WALWALASAFI 

- Your both projects have spacing issues, ask your partner to help you with that.

**Comment by WALWALASAFI** [comment]
@ahzamir
sir is it cleaned now, i added some style and linked it style.css with index.html,but when i see in google. it didnt link!

**Comment by ahzamir** [comment]
> @ahzamir sir is it cleaned now, i added some style and linked it style.css with index.html,but when i see in google. it didnt link!

- The issue is in your CSS file naming your CSS file name is `style.css.css`  which is wrong rename your CSS file and change it to `style.css`.

**Review by ahzamir — APPROVED** [review]
### Hi @WALWALASAFI 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Tribute Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by WALWALASAFI** [comment]
much obliged dear sir
oh my gosh, i got so happy 
i am working on technical-documention project.
inshallah this project wont have any problem.

**Pull request merged**